src/wrap.js

```diff
@@ -13,7 +13,7 @@ export default function wrap (el) {
     return new ReactTestWrapper(el)
   }
 
-  if (el && el._root && el.options) {
+  if (el && el.cheerio && el.options) {
```

Comment:
This doesn't need a fallback (to `_root` for old cheerio versions)?

Reply:
It looks `Cheerio.prototype.cheerio` has been around since `0.19.0`: https://github.com/cheeriojs/cheerio/blob/0.19.0/lib/cheerio.js#L99